File: src/test/scala/cern/colt/matrix/tdouble/impl/SparseCCMDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.SparseCCMDoubleMatrix2D

class SparseCCMDoubleMatrix2DViewTest(arg0: String) extends SparseCCMDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseCCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseCCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseCCMDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tdouble/impl/SparseCCMDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class SparseCCMDoubleMatrix2DViewTest extends SparseCCMDoubleMatrix2DTest {

    public SparseCCMDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCMDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
